pymeasure/instruments/rohdeschwarz/fsl.py

```diff
@@ -113,7 +113,7 @@ def __init__(self, adapter, **kwargs):
     continuous_sweep = Instrument.control(
         "INIT:CONT?",
         "INIT:CONT %s",
-        "Continuous (True) or single sweep (False)",
+        "Continuous (True) ophase_noise_tracer single sweep (False)",
```

Comment:
Guess a typo :)

Reply:
Resolved via [1d49a15](https://github.com/bleykauf/pymeasure/pull/4/commits/1d49a15a1c3277b3bcad59f81feb01e81f3c4836)

Reply:
Still says `phase_noise_tracer` instead of `or`